private/pkg/app/app.go

```diff
@@ -39,7 +39,7 @@ type EnvContainer interface {
 	// GetEnvBoolValue gets and parses the environment variable bool value for the key.
 	//
 	// Returns false and the parse error if the key is not set or the value is empty.
-	GetEnvBoolValue(key string) (bool, error)
+	GetEnvBoolValue(key string, defaultVal bool) (bool, error)
```

Comment:
I didn't know this function existed on `EnvContainer` - separately, this function should be removed, but for this PR, don't add to it more. If we want something like this, then `EnvBool(envContainer EnvContainer, key string) (bool, bool)` or `EnvBool(envContainer EnvContainer, key string, default bool) bool` would be acceptable as a top-level function

Reply:
(which, if you want to do, you might as well delete this function from EnvContainer and then add EnvBool heh)

Reply:
I added the method to begin with because `container.Env` returns an empty string when a key is not set which means that callers will always need to check for an empty string before passing it to `strconv.ParseX`, then handle the parse errors.

Reply:
Yea this is more philosophy of this package - the containers are meant to be the absolute minimal thing you need to represent what they are containing, if you want to do helpers on top of them, add them as global functions. See `Args` as an example - just being consistent here basically

Reply:
👍